fused-effects.cabal

```diff
@@ -19,12 +19,15 @@ extra-source-files:
 tested-with:         GHC == 8.2.2
                    , GHC == 8.4.4
                    , GHC == 8.6.2
+                   , GHC == 8.8.1
 
 common common
   default-language:    Haskell2010
   ghc-options:         -Weverything -Wno-missing-local-signatures -Wno-missing-import-lists -Wno-implicit-prelude -Wno-safe -Wno-unsafe -Wno-name-shadowing -Wno-monomorphism-restriction -Wno-missed-specialisations -Wno-all-missed-specialisations
   if (impl(ghc >= 8.6))
     ghc-options:       -Wno-star-is-type
+  if (impl(ghc >= 8.8))
+    ghc-options:       -Wno-missing-deriving-strategies
```

Comment:
This is new in 8.8.1—it warns on every single `deriving` site.

Reply:
😍